assets/styles/settings/_config.fonts.scss

```diff
@@ -59,3 +93,12 @@
     $value: map-get($font-families, $font-family);
     @return $value;
 }
+
+
+// Include fonts
+// ==========================================================================
+
+// Font directory
+$font-dir: "../fonts/";
+
+@include font-faces($font-faces, $font-dir);
```

Comment:
I think this should remain in `_document.scss`. The configuration files, like tools, should not output CSS.

Reply:
True, done !